src/snapshot.rs

```diff
@@ -23,6 +23,25 @@ pub struct Snap {
     pub ledger_entries: OrdMap<LedgerKey, LedgerEntry>,
 }
 
+pub fn get_default_ledger_info() -> LedgerInfo {
+    LedgerInfo {
+        protocol_version: 19,
+        sequence_number: 0,
+        timestamp: 0,
+        network_id: vec![0u8],
```

Comment:
We should change this field to network_passphrase.

Reply:
I'll open an env PR to fix this now

Reply:
Done